frontend/src/api/Api.js

```diff
@@ -13,6 +13,7 @@ Api.interceptors.response.use(
       sessionStorage.clear();
       window.location.href = '/';
     }
+    return Promise.reject(error);
```

Comment:
I added this line because at some cases, when client receives error response from backend, then it is not delivered from this interceptor to the place where request was made. So, all the catch blocks did not work. This line fixes the problem.